mkosi/backend.py

```diff
@@ -225,17 +243,17 @@ class ManifestFormat(Parseable, enum.Enum):
 
 
 class PartitionIdentifier(enum.Enum):
-    esp        = 'esp'
-    bios       = 'bios'
-    xbootldr   = 'xbootldr'
-    root       = 'root'
-    swap       = 'swap'
-    home       = 'home'
-    srv        = 'srv'
-    var        = 'var'
-    tmp        = 'tmp'
-    verity     = 'verity'
-    verity_sig = 'verity-sig'
+    esp        = "esp"
+    bios       = "bios"
+    xbootldr   = "xbootldr"
+    root       = "root"
+    swap       = "swap"
+    home       = "home"
+    srv        = "srv"
+    var        = "var"
+    tmp        = "tmp"
+    verity     = "verity"
+    verity_sig = "verity-sig"
```

Comment:
Bumped into this usage of single quotes and as this has been a topic in quite a few of my PR's, I thought it couldn't hurt fixing it quickly